src/ghost.js

```diff
@@ -68,7 +68,16 @@ class GhostThread extends EventEmitter {
    * @return    {Array<Object>}      users online
    */
   async getPosts () {
-    return [...this._backlog].map(msg => JSON.parse(msg)).sort((p1, p2) => p1.timestamp - p2.timestamp)
+    const _12hrInSecs = 12 * 60 * 60
```

Comment:
Move this constant to the top of the file. Usually constants are named with all capital letters. 
I would name it someting like: 
```js
const POST_AGE_LIMIT = 12 * 60 * 60 // 12 hours
```